allennlp/models/archival.py

```diff
@@ -129,12 +134,17 @@ def load_archive(archive_file: str, cuda_device: int = -1, overrides: str = "")
         overrides = json.dumps(combined_hocon)
 
     # Load config
-    config = Params.from_file(os.path.join(serialization_dir, _CONFIG_NAME), overrides)
+    config = Params.from_file(os.path.join(serialization_dir, CONFIG_NAME), overrides)
```

Comment:
Is this backwards compatible if `archive_file` is a directory rather than a tar.gz file? I'm not sure it is.

Reply:
Also if this is a breaking change for the release that's fine, it's just quite a big thing to break accidentally so I was just checking

Reply:
Hi @DeNeutoy I don't understand.  What exactly are you worried about being backwards incompatible?  The line you referenced only switches from a private constant to a public one.